### ISSUE TITLE
Problem to access to profile page

### ISSUE THREAD

**Issue by WaelSan** [bigbluebutton/greenlight#1337]
There is a regression to access to profile page.
This issue didn't appear with old greenlight version (2 weeks ago).
But today I tried to install greenlight and I found this problem.

![image](https://user-images.githubusercontent.com/29982639/79674197-56b09e80-81e9-11ea-919b-5761531dd75a.png)

**Comment by testerik** [comment]
Can confirm this. From production.log:

```
method=GET path=/b/u/gl-ktcqznuvbemh/edit format=html controller=UsersController action=edit status=500 error='ActionView::Template::Error: undefined method `[]' for nil:NilClass' duration=63.03 view=0.00 db=5.28 host=greenlight 
ActionView::Template::Error (undefined method `[]' for nil:NilClass): 
36:     <%= f.text_field :provider, class: "form-control", readonly: "" %>
37:     <br>
38:     <%= f.label t("settings.account.language"), class: "form-label" %>
39:     <%= f.select :language, language_options, {}, { class: "form-control custom-select" } %>
40: 
41:     <% current_user_role = current_user.highest_priority_role %>
42:     <br> 
app/helpers/users_helper.rb:50:in `block in language_options'
app/helpers/users_helper.rb:48:in `each'
app/helpers/users_helper.rb:48:in `language_options'
app/views/users/components/_account.html.erb:39:in `block in _app_views_users_components__account_html_erb__1922620160339889164_47060312780400'
app/views/users/components/_account.html.erb:16:in `_app_views_users_components__account_html_erb__1922620160339889164_47060312780400'
app/views/users/components/_setting_view.html.erb:24:in `block in _app_views_users_components__setting_view_html_erb__1462585977136171657_47060312797040'
app/views/users/components/_setting_view.html.erb:16:in `_app_views_users_components__setting_view_html_erb__1462585977136171657_47060312797040'
app/views/users/edit.html.erb:32:in `_app_views_users_edit_html_erb___396747478642196272_47060312847960' 
INFO: [8d57ac98-d37b-4690-a171-f04acdca385c] method=GET path=/b/500 format=html controller=ErrorsController action=internal_error status=500 duration=32.31 view=25.67 db=0.10 host=greenlight
```

**Comment by pajowu** [comment]
Problem seems to be this commit: https://github.com/bigbluebutton/greenlight/commit/3b8b1abb59f56c4ee4b4316d559c2205b7a808ee

tl;dr
The commit added a new language whichs short code is not known to the greenlight. This crashed when the list of available languages is shown.

long version:

greenlight automatically adds the "hy" language to `Rails.configuration.i18n.available_locales` ([here](https://github.com/bigbluebutton/greenlight/blob/b2f2e73bbc2481cd2661cc8df3e24a6d8a5d6d13/config/initializers/languages.rb#L12)) and later uses it [here](https://github.com/bigbluebutton/greenlight/blob/b2f2e73bbc2481cd2661cc8df3e24a6d8a5d6d13/app/helpers/users_helper.rb#L50) to render the list of available languages. But the [`i18n-language-mapping`](https://github.com/blindsidenetworks/i18n-language-mapping) gem does not support this language (https://github.com/blindsidenetworks/i18n-language-mapping/blob/master/lib/i18n/language/mapping.rb)

Quickfix: Delete the `hy.yml` file and redeploy greenlight
Proper fix: Add `hy` to the `i18n-language-mapping` gem and/or properly handling missing language mappings

**Comment by GremL1N** [comment]
Also we have `config.i18n.available_locales = [:en]` in `config/application.rb' which  redefine by `config/initializers/languages.rb' and not working. I thing if we could use `config.i18n.available_locales` to limit set of languages, we may keep any file in `config/locales`. Am i right?

**Comment by farhatahmad** [comment]
Fixed by #1361

**Comment by GremL1N** [comment]
need to restore hy.yml otherwise `controller=ErrorsController action=internal_error status=500 error='I18n::InvalidLocaleData: can not load translations from /usr/src/app/config/locales/hy.yml: expects it to return a hash, but does not'`
